dev/com.ibm.ws.microprofile.faulttolerance.cdi.1.1.services/bnd.bnd

```diff
@@ -0,0 +1,39 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.cdi.1.1.services
+Bundle-SymbolicName: com.ibm.ws.microprofile.faulttolerance.cdi.1.1.services
+Bundle-Description: MicroProfile Fault Tolerance CDI Integration, version ${bVersion} services
+
+Private-Package: \
+    com.ibm.ws.microprofile.faulttolerance11.cdi.config
+    
+Import-Package: \
+    org.eclipse.microprofile.config.*;version="[1.0,2)",\
```

Comment:
FT 1.1 depends on Config 1.3 ONLY ... should that range be `[1.3,2)`?

Reply:
is the import even needed here?

Reply:
No, because it's the package version and the package has no major changes so the 1.3 package is still version 1.0.1.